src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -51,6 +51,8 @@ api_platform:
         Sylius\Bundle\ApiBundle\Exception\ProvinceCannotBeRemoved: 422
         Sylius\Bundle\ApiBundle\Exception\ShippingMethodCannotBeRemoved: 422
         Sylius\Bundle\ApiBundle\Exception\ZoneCannotBeRemoved: 422
+        Sylius\Component\Core\Inventory\Exception\NotEnoughUnitsOnHandException: 422
+        Sylius\Component\Core\Inventory\Exception\NotEnoughUnitsOnHoldException: 422
```

Comment:
Is there any possibility these will be thrown on the shop's side?

Reply:
To my current knowledge, no, because the `PaymentPreCompleteListener` should have been triggered on the UI (admin/shop) beforehand

Reply:
I meant the shop's API, but we don't have paying covered atm so 🤷